cultcargo/builder/cargo-manifest.yml

```diff
@@ -115,21 +115,28 @@ images:
     versions:
       '1.2.0':
         package: msutils==1.2.0
-      latest: 
+      latest:
         package: msutils
       master:
         package: git+https://github.com/SpheMakh/msutils
 
   quartical:
     versions:
       base:
-        package: git+https://github.com/ratt-ru/QuartiCal@stimelation
+        package: git+https://github.com/ratt-ru/QuartiCal@v0.2.1-dev
```

Comment:
Shouldn't this just the latest PyPI release?

Reply:
I believe so, now that 0.2.2 is out.